integrations/azure-devops/azure_devops/gitops/port_app_config.py

```diff
@@ -0,0 +1,17 @@
+from port_ocean.core.handlers.port_app_config.models import (
+    PortAppConfig,
+    ResourceConfig,
+)
+from .file_entity_processor import GitSelector
+from pydantic import Field
+from typing import List
+
+
+class GitResourceConfig(ResourceConfig):
+    selector: GitSelector
+
```

Comment:
if it only applies to specific kinds then you can sure to validate it easily using pydantic